### PR TITLE
Create miva

### PR DESCRIPTION
**Pull request by powermyanalytics**

### CODE REVIEW

File: organizations/miva [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (pfrisella) [inline organizations/miva:1]: file is missing .json extension

**Review by pfrisella — CHANGES_REQUESTED** [review]

File: organizations/miva (hunk 1 above)
Comment (powermyanalytics) [reply organizations/miva:1]: Can you please add to the end?  I forgot...

Comment (powermyanalytics) [reply organizations/miva:1]: Or please let me know how I add the extension...not sure how to edit this.

**Comment by pfrisella** [comment]
Resolved in another PR

**Pull request closed without merging**